docker/install-system-dependencies

```diff
@@ -83,6 +78,7 @@ linux_generic_av_packages=(
 	libasound2-dev:native
 	libaudio-dev:native
 	libgl1-mesa-dev:native
+ 	libglu1-mesa-dev:native
```

Comment:
This is not a big deal, but we shouldn't need GLU. Previously I had to add some define like `GLEW_NO_GLU` to prevent a dependency header from trying to include it.

Reply:
Done.